.github/workflows/android.yml

```diff
@@ -0,0 +1,49 @@
+name: Android CI
+
+on: [push, pull_request]
+
+defaults:
+  run:
+    working-directory: android
```

Comment:
If you want set a working directory you should do this one the specific step you are interested in **or** simply `cd android` before you run the gradle command.

Reply:
Yeah it seems to work the same both ways?

How do I use chmod +x gradlew in a proper way? Having a hard time finding a resource explaining this commad. Where do I use it?

Reply:
Nevermind, Think I got it.